src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -202,48 +201,43 @@ const MapView: React.FC<MapViewProps> = ({ blocks, neighborhoods }) => {
           }
           return name;
         }
-        */
+
         // Loads the objects into the layer
         neighborhoodsLayer.addGeoJson(neighborhoods);
-        /*
-          () => {
-            // For each feature in neighbourhoodsLayer, add a marker
-            // We need to do it here as the GeoJson needs to load first
-            neighborhoodsLayer.forEach((feature) => {
-              // If you want, check here for some constraints.
-              const marker = new google.maps.Marker({
-                map,
-                draggable: false,
-                label: {
-                  color: 'white',
-                  fontWeight: 'bold',
-                  text: shortHand(feature.getProperty('Name')),
-                },
-                // Removed the icon here, only text on map.
-                icon: {
-                  labelOrigin: new google.maps.Point(11, 50),
-                  url: '',
-                  size: new google.maps.Size(22, 40),
-                },
-                position: {
-                  lat: feature.getProperty('Latitude'),
-                  lng: feature.getProperty('Longitude'),
-                },
-              });
-              markersArray.push(marker);
-              marker.setMap(map);
-            });
-          },
-        );
-        */
+        neighborhoodsLayer.forEach((feature) => {
+          // For each feature in neighbourhoodsLayer, add a marker
+          // We need to do it here as the GeoJson needs to load first
+          // If you want, check here for some constraints.
+          const marker = new google.maps.Marker({
+            map,
+            draggable: false,
+            label: {
+              color: 'white',
+              fontWeight: 'bold',
+              text: shortHand(feature.getProperty('name')),
+            },
+            // Removed the icon here, only text on map.
+            icon: {
+              labelOrigin: new google.maps.Point(11, 50),
+              url: '',
+              size: new google.maps.Size(22, 40),
+            },
+            position: {
+              lat: feature.getProperty('lat'),
+              lng: feature.getProperty('lng'),
+            },
+          });
+          markersArray.push(marker);
+          marker.setMap(map);
+        });
 
         // Sets the style of the layer to green shades based on property values with white outline
         function setNeighborhoodsStyle(v: boolean) {
           neighborhoodsLayer.setStyle((feature) => {
             return {
               fillColor: 'green',
               fillOpacity:
-                (feature.getProperty('Neighborhood_ID') / 100) * 2 + 0.1, // TODO: replace this with completion percentage
+                (feature.getProperty('neighborhood_id') / 100) * 2 + 0.1, // TODO: replace this with completion percentage
```

Comment:
We have the 'completionPerc' on the neighborhood features, we should be able to use that instead of id.

Reply:
Spoke to Sofie about this and we should leave it as todo for now